exist-core/src/main/java/org/exist/http/urlrewrite/XQueryURLRewrite.java

```diff
@@ -179,7 +179,7 @@ protected void service(final HttpServletRequest request, final HttpServletRespon
         } else {
             // Secondly try basic authentication
             final String auth = request.getHeader("Authorization");
-            if (auth != null) {
+            if (auth != null && auth.startsWith("Basic ")) {
```

Comment:
Same as before

Reply:
Good spot @reinhapa! I have now added the case-insensitive match